### PR TITLE
[dv/alert_handler] V2 checklist

### PR DESCRIPTION
**Pull request by cindychip**
This PR completes alert_handler V2 checklist.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

File: hw/ip_templates/alert_handler/data/alert_handler.prj.hjson [hunk 1]
```diff
@@ -11,7 +11,7 @@
     version:            "1.0",
     life_stage:         "L1",
     design_stage:       "D2",
-    verification_stage: "V1",
+    verification_stage: "V2",
```
Comment (cindychip) [inline hw/ip_templates/alert_handler/data/alert_handler.prj.hjson:14]: Thanks everyone for attending the review meeting.
Here is the [notes](https://docs.google.com/document/d/1s9lnLFccW0Y479oxogVnm2-I3HgPlt7UcbqJXHdTlps/edit?resourcekey=0-h91JxZynFGWCsOZB01qfMQ) from the meeting.

The TODOs are listed below:

- [ ] Check with security group if missing a recoverable alert during alert_init is a problem
        If so, add a V3 test: sending a alert_init while receiving recoverable alert 

      Added a V3 item: https://github.com/lowRISC/opentitan/pull/9723 

- [ ] Check coverage is converging
image

- [ ] Add an assertion to check the alert/esc counter is saturated after reaching the maximum value. Also check if it’s converged in FPV (we can change the parameter to reduce the size for ping)

   Added assertion: https://github.com/lowRISC/opentitan/pull/9715 

- [ ] Do the regwen coverage in RAL model

    Filed issue: https://github.com/lowRISC/opentitan/issues/9673 

- [ ] Check if the checklist has any missing item

   Checked and no missing item found!

- [ ] Check if we can remove exclusion at *_clear_regwen

  Done with PR: https://github.com/lowRISC/opentitan/pull/9674

Comment (weicaiyang) [reply hw/ip_templates/alert_handler/data/alert_handler.prj.hjson:14]: This checklist is awesome. Thanks

**Review by weicaiyang — APPROVED** [review]
LGTM, thanks

**Review by matutem — APPROVED** [review]
LGTM

**Review by msfschaffner — APPROVED** [review]

**Pull request merged**